lib/src/edx_cookiecutter_lib/post_code.py

```diff
@@ -62,3 +62,11 @@ def post_gen_project(extra_context):
 
     # Post build fixes
     write_main(['pylintrc'])
+
+    # Cookiecutters are right for the openedx organization. Other orgs might need
+    # adjustments.
+    org = extra_context["github_org"]
+    if org != "openedx":
+        print("*" * 78)
+        print(f"Since your repo will be in the {org} organization, you may need")
+        print("to adjust the contents of the repo. Check with your organization.")
```

Comment:
Maybe give examples of what might need to be changed? For example:

```suggestion
        print("to adjust the contents of the repo, such as licenses and email")
        print("addresses. Check with your organization.")
```

Reply:
Good idea!